README.md

```diff
@@ -47,10 +47,13 @@ chmod +x ./kind-darwin-amd64
 mv ./kind-darwin-amd64 /some-dir-in-your-PATH/kind
 ```
 
-To use kind, you will need to [install docker].  
-Once you have docker running you can create a cluster with `kind create cluster`  
+To use kind, you will need to [install docker].
+Once you have docker running you can create a cluster with `kind create cluster`
 To delete your cluster use `kind delete cluster`
 
+**NOTE**: Your user running `kind` needs permission to access docker socket,
```

Comment:
I don't think this warning belongs in the beginning of the readme. This should be in the user guide or really in the known issues doc.

I think it's reasonable to assume that you need access to docker in order to use "Kubernetes in Docker" ...

Reply:
Yes, I agree. 
let me move it to known issues doc :)

I will point to the origin issue https://github.com/kubernetes-sigs/kind/issues/713#issue-468988436

>I got an error "Error: could not list clusters: invalid output when listing nodes: WARNING: Error loading config file: /home/necuser/.docker/config.json: open /home/necuser/.docker/config.json: permission denied"

Reply:
SGTM

Reply:
Updated. PTAL.